test/runtime/configMatters/comm/AtomicsAPI.chpl

```diff
@@ -16,6 +16,7 @@ proc testAtomicBool(a, ref i, ref b) {
 
   writeln("Testing 'atomic bool':");
                                                     writea ("init    ", a);
+                var initval : a.type = true;        writeai("init=   ", a, initval);
```

Comment:
Add something like `var initval2: a.type = a` or whatever to test the `other: RAtomicBool` case too?

Reply:
And maybe `var initval3 = initval; assert(initval3.type == atomic basetype; ...`?